R/palaeorotate.R

```diff
@@ -437,6 +439,8 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
       chk <- seq(from = 0, to = nr + chunks, by = chunks)
```

Comment:
I think everything from line 426 down should be indented once? Then everything from line 445 should be indented an extra time?